web/src/cypress/integration/profile.ts

```diff
@@ -54,12 +71,20 @@ function testProfile(screen: ScreenFormat) {
       cy.get('input[name=name]')
         .clear()
         .type(name)
+      cy.get('input[name=value]')
+        .clear()
+        .type(value)
       cy.get('button[type=submit]').click()
 
       cy.get('ul[data-cy=contact-methods]').should(
         'contain',
         `${name} (${cm.type})`,
       )
+      cy.get('ul[data-cy="contact-methods"] li')
+        .contains(`${name} (${cm.type})`)
```

Comment:
Can we try simplifying this a bit?

Maybe this would work:
```js
cy.get('ul[data-cy=contact-methods]')
  .contains('li', '${name} (${cm.type})')
  .find('[data-cy=cm-disabled]')
```

- Get the list
- Find the `li` that contains the text
- Find the `cm-disabled` element within the `li`